src/leap/mail/imap/account.py

```diff
@@ -46,107 +46,56 @@ def _debugProfiling(result, cmdname, start):
 
 
 #######################################
-# Soledad Account
+# Soledad IMAP Account
 #######################################
 
+# XXX watchout, account needs to be ready... so we should maybe return
+# a deferred to the IMAP service when it's initialized
```

Comment:
Instead of this comment we could maybe document `callWhenReady` method or put this warning in the class docstring.